package.json

```diff
@@ -1,222 +1,250 @@
-      }
-    ],
-    "grammars": [
-      {
-        "language": "code-runner-output",
-        "scopeName": "code-runner.output",
-        "path": "./syntaxes/code-runner-output.tmLanguage"
-      }
-    ]
-  },
-  "scripts": {
-    "vscode:prepublish": "node ./node_modules/vscode/bin/compile",
-    "compile": "node ./node_modules/vscode/bin/compile -watch -p ./",
-    "postinstall": "node ./node_modules/vscode/bin/install"
-  },
-  "dependencies": {
-    "applicationinsights": "^0.17.1",
-    "tree-kill": "^1.1.0"
-  },
-  "devDependencies": {
-    "typescript": "^1.8.5",
-    "vscode": "^0.11.0"
-  }
+	"name": "code-runner",
+	"displayName": "Code Runner",
+	"description": "Run code snippet/file for AppleScript, BASH/SH, Batch/CMD, C/C++, C#, Clojure, CoffeeScript, Crystal, Elixir, F#, Go, Groovy, Java, JavaScript, Julia, LUA, OCaml, Perl, PHP, PowerShell, Python, R, Ruby, Scala, Swift, TypeScript, VB Script and VB.NET.",
+	"version": "0.6.8",
+	"publisher": "formulahendry",
+	"icon": "images/logo.png",
+	"engines": {
+		"vscode": "^1.6.0"
+	},
+	"categories": [
+		"Languages",
+		"Other"
+	],
+	"keywords": [
+		"javascript",
+		"perl",
+		"php",
+		"python",
+		"ruby"
+	],
+	"bugs": {
+		"url": "https://github.com/formulahendry/vscode-code-runner/issues",
+		"email": "formulahendry@gmail.com"
+	},
+	"homepage": "https://github.com/formulahendry/vscode-code-runner/blob/master/README.md",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/formulahendry/vscode-code-runner.git"
+	},
+	"activationEvents": [
+		"onCommand:code-runner.runCode",
+		"onCommand:code-runner.runCodeByLanguage",
+		"onCommand:code-runner.runCustomCommand"
+	],
+	"main": "./out/src/extension",
+	"contributes": {
+		"commands": [
+			{
+				"command": "code-runner.runCode",
+				"title": "Code Runner: Run Code"
+			},
+			{
+				"command": "code-runner.runCodeByLanguage",
+				"title": "Code Runner: Run Code By Language"
+			},
+			{
+				"command": "code-runner.runCustomCommand",
+				"title": "Code Runner: Run Custom Command"
+			},
+			{
+				"command": "code-runner.stop",
+				"title": "Code Runner: Stop Code Run"
+			}
+		],
+		"keybindings": [
+			{
+				"command": "code-runner.runCode",
+				"key": "ctrl+alt+n"
+			},
+			{
+				"command": "code-runner.runCodeByLanguage",
+				"key": "ctrl+alt+j"
+			},
+			{
+				"command": "code-runner.runCustomCommand",
+				"key": "ctrl+alt+k"
+			},
+			{
+				"command": "code-runner.stop",
+				"key": "ctrl+alt+m"
+			}
+		],
+		"menus": {
+			"editor/context": [
+				{
+					"when": "!inOutput",
+					"command": "code-runner.runCode",
+					"group": "navigation"
+				},
+				{
+					"when": "inOutput",
+					"command": "code-runner.stop",
+					"group": "stop-code-run"
+				}
+			]
+		},
+		"configuration": {
+			"type": "object",
+			"title": "Run Code configuration",
+			"properties": {
+				"code-runner.executorMap": {
+					"type": "object",
+					"default": {
+						"AppleScript": "osascript.exe",
```

Comment:
Do not change the id list, they come from the [Language Identifiers](https://code.visualstudio.com/docs/languages/identifiers)